src/components/UserActions/ClickedDocumentList.ts

```diff
@@ -124,19 +113,22 @@ export class ClickedDocumentList extends Component {
     }
 
     private render() {
-        new ExpandableList<IQueryResult>(this.element, this.sortedDocumentsList, {
+        new ExpandableList<UserAction>(this.element, this.sortedDocumentsList, {
             maximumItemsShown: this.sortedDocumentsList.length,
             minimumItemsShown: this.options.numberOfItems,
-            transform: (result) => {
-                QueryUtils.setStateObjectOnQueryResult(this.queryStateModel.get(), result);
-                QueryUtils.setSearchInterfaceObjectOnQueryResult(this.searchInterface, result);
-                return (<Promise<HTMLElement>>this.options.template.instantiateToElement(result, {
+            transform: (action) => {
+                QueryUtils.setStateObjectOnQueryResult(this.queryStateModel.get(), action.document);
+                QueryUtils.setSearchInterfaceObjectOnQueryResult(this.searchInterface, action.document);
+                return (<Promise<HTMLElement>>this.options.template.instantiateToElement(action.document, {
                     wrapInDiv: true,
                     checkCondition: true,
                     currentLayout: 'list',
                     responsiveComponents: this.searchInterface.responsiveComponents,
                 })).then((element) => {
-                    Initialization.automaticallyCreateComponentsInsideResult(element, result);
+                    Initialization.automaticallyCreateComponentsInsideResult(element, action.document);
+                    if (action.raw.origin_level_1) {
```

Comment:
The issue description says:

> Add a tooltip on each item of the session summary to display the origin_level_1 or the search-hub of the items

But, we are only checking for `origin_level_1` here. Is it because `origin_level_1` defaults on the search hub?

Reply:
origin level 1 is the same thing as the searchhub yes, I put searchhub here because the terms are interchangeable most of the time people don't know that searchhub is actually originLevel1 as far as analytics is concerned